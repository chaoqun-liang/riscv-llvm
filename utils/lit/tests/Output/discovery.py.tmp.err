lit.py: discovery.py:45: note: loading suite config '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/exec-discovery/lit.site.cfg'
lit.py: LitConfig.py:97: note: load_config from '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/discovery/lit.cfg'
lit.py: TestingConfig.py:103: note: ... loaded config '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/discovery/lit.cfg'
lit.py: TestingConfig.py:103: note: ... loaded config '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/exec-discovery/lit.site.cfg'
lit.py: discovery.py:118: note: resolved input '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/../tests/Inputs/exec-discovery' to 'top-level-suite'::()
lit.py: discovery.py:96: note: loading local config '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/discovery/subdir/lit.local.cfg'
lit.py: TestingConfig.py:103: note: ... loaded config '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/discovery/subdir/lit.local.cfg'
lit.py: discovery.py:45: note: loading suite config '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/discovery/subsuite/lit.cfg'
lit.py: TestingConfig.py:103: note: ... loaded config '/home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/discovery/subsuite/lit.cfg'
