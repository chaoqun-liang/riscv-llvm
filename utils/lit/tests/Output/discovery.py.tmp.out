-- Test Suites --
  exec-discovery-in-tree-suite - 1 tests
    Source Root: /home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/exec-discovery-in-tree
    Exec Root  : /home/shared/riscv128/riscv-tools/riscv-llvm/utils/lit/tests/Inputs/exec-discovery-in-tree/obj
-- Available Tests --
  exec-discovery-in-tree-suite :: test-one.txt
